### PR TITLE
Added ical export for events

### PR DESCRIPTION
**Pull request by xavismeh**
Follow up of #342 
Fixes https://github.com/EnMarche/en-marche.fr/issues/295

### CODE REVIEW

File: src/AppBundle/Controller/EventController.php [hunk 1]
```diff
@@ -36,6 +37,24 @@ public function showAction(Event $event): Response
...
+     *   path="/ical",
+     *   name="app_committee_event_export_ical"
+     * )
+     * @Method("GET")
+     */
+    public function exportIcalAction(Event $event): Response
+    {
+        return new Response(
+            $this->get('jms_serializer')->serialize($event, 'ical'),
+            Response::HTTP_OK, [
```
Comment (tgalopin) [inline src/AppBundle/Controller/EventController.php]: You should add a line break here.

File: src/AppBundle/Controller/EventController.php [hunk 2]
```diff
@@ -36,6 +37,24 @@ public function showAction(Event $event): Response
     }
 
     /**
+     * @Route(
+     *   path="/ical",
```
Comment (tgalopin) [inline src/AppBundle/Controller/EventController.php]: Can you use a single line for the route?

File: src/AppBundle/Serializer/EventICalHandler.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    /**
+     * @param \DateTime $date
+     *
+     * @return string
+     */
+    private function formatDate(\DateTime $date): string
+    {
+        return $date->format('Ymd\THis');
```
Comment (tgalopin) [inline src/AppBundle/Serializer/EventICalHandler.php]: You should use http://php.net/manual/fr/class.datetime.php#datetime.constants.atom

Comment (xavismeh) [inline src/AppBundle/Serializer/EventICalHandler.php]: This format does not match according to ical specifications and the testing tool I've used to validate file format recommended to use this

Comment (tgalopin) [inline src/AppBundle/Serializer/EventICalHandler.php]: Okay :)

**Comment by xavismeh** [comment]
@tgalopin I've added a simple unit test as requested + merged current master branch into this

**Comment by tgalopin** [comment]
Closed by https://github.com/EnMarche/en-marche.fr/pull/569.

Thanks @xavismeh !

**Pull request closed without merging**